packages/plugin-team/src/team.js

```diff
@@ -0,0 +1,286 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import './decrypter';
+import './encrypter';
+
+import {find, pick, uniq} from 'lodash';
+import {SparkPlugin} from '@ciscospark/spark-core';
+
+const Team = SparkPlugin.extend({
+  namespace: `Team`,
+
+  /**
+   * Move an existing group conversation into a team.
+   * @param {TeamObject} team
+   * @param {ConversationObject} conversation
+   * @returns {Promise} Resolves with the add activity
+   */
+  addConversation(team, conversation) {
+
```

Comment:
I just added the stubs for all the action methods. They will be implemented in Teams 2 PR